glotaran/builtin/models/kinetic_spectrum/test/test_spectral_irf.py

```diff
@@ -280,7 +280,7 @@ def test_spectral_irf(suite):
     assert np.array_equal(dataset["spectral"], resultdata["spectral"])
     assert dataset.data.shape == resultdata.data.shape
     assert dataset.data.shape == resultdata.fitted_data.shape
-    assert np.allclose(dataset.data, resultdata.fitted_data, atol=1e-15)
+    assert np.allclose(dataset.data, resultdata.fitted_data, atol=1e-14)
```

Comment:
Yes, this tiny raising of absolute tolerance is acceptable. 

But this test needs to be revised anyways, when I do that I will have another look.